### PR TITLE
feat(true-up): Test cases for true-up: Support for customers with no connectivity

### PR DESCRIPTION
**Pull request by yasserfaraazkhan**
feat(true-up): Test cases for true-up: Support for customers with no connectivity

closes: [MM-49440](https://mattermost.atlassian.net/browse/MM-49440)

Epic: [MM-47851](https://mattermost.atlassian.net/browse/MM-47851)

<a href="https://gitpod.io/#https://github.com/mattermost/mattermost-test-management/pull/50"></a>

### CODE REVIEW

**Review by lindy65 — APPROVED** [review]
@yasserfaraazkhan - the tests look good to me but you might want to also get a review from @stevemudie as I'm not that familiar with Cloud processes from the customer's perspective.

**Review by ConorMacpherson — APPROVED** [review]

**Review by stevemudie — APPROVED** [review]
These look good @yasserfaraazkhan!

File: data/test-cases/self-hosted/true-up-support/test-1.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+**Step 1**
+
+<https://mattermost.atlassian.net/browse/MM-47851>
+
+Pre-requisites:
+
+- On an Non Air gapped instance, the True up Review section should be visible 15 days before end of quarter.
+- Customers will have 15 days grace period after end of quarter to submit the report.
+  eg: The review section will be visible on March 16th till April 15th. Once Admin Submits, the section should not be displayed.
```
Comment (johndavidlugtu) [inline data/test-cases/self-hosted/true-up-support/test-1.md:43]: Just to highlight that if there's no action from the customer 15 days after the end of the calendar quarter (e.g. April 15), the true-up review section should disappear

**Pull request merged**